### ISSUE TITLE
make a new release

### ISSUE THREAD

**Issue by anarcat** [cryptodotis/irssi-otr#64]
1.0.1 has critical issues regarding compatibility with irssi (#60, #62) and a bunch more small issues, yet there was no release published. can a tag be pushed now please? i had to package a git snapshot in debian, which makes things needlessly confusing...

**Comment by varac** [comment]
That would be awesome :dancing_men:

**Comment by geor-g** [comment]
...actually, it would be amazing. :1st_place_medal:

**Comment by micah** [comment]
There isn't much of a need for making a new release of this as otr is being integrated into irssi directly.

https://github.com/irssi/irssi/tree/ahf/otr
https://github.com/irssi/irssi/commit/18a37d826c313ad4d01f2f96d14d66b350347d42
https://github.com/irssi/irssi/pull/854

**Comment by claucece** [comment]
I agree with @micah . OTR support is now integrated with irssi: https://github.com/irssi/irssi/pull/854
